src/System.Management.Automation/DscSupport/CimDSCParser.cs

```diff
@@ -73,7 +73,8 @@ public static object ConvertCimInstanceToObject(Type targetType, CimInstance ins
                     MemberInfo[] memberInfo = targetType.GetMember(property.Name, BindingFlags.Public | BindingFlags.Instance);
 
                     // verify property exists in corresponding class type
-                    if (memberInfo == null || memberInfo.Length > 1 || !(memberInfo[0] is PropertyInfo || memberInfo[0] is FieldInfo))
+                    if (memberInfo == null || memberInfo.Length > 1 ||
+                        (memberInfo[0] is not PropertyInfo && memberInfo[0] is not FieldInfo))
```

Comment:
```suggestion
                    if (memberInfo == null
                        || memberInfo.Length > 1
                        || (memberInfo[0] is not PropertyInfo && memberInfo[0] is not FieldInfo))
```

Newer portions of the codebase tend towards this format, we should probably keep it consistent as we make edits. 🙂

Reply:
@vexx32 I was trying mostly to keep existing code style, but it appears your suggested change is the more modern format. Also the wrap binary expressions refactoring places the operator on a new line: https://docs.microsoft.com/visualstudio/ide/reference/wrap-indent-align-refactorings?view=vs-2019#wrap-binary-expressions